code/clue.js

```diff
@@ -1,59 +1,237 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+const Marcel = {
+  firstName: 'Marcel',
+  lastName: 'the Monkey',
```

Comment:
Congratulations for making this project original :)
Just one thing - we shouldn't declare variables with capital letters. This approach is reserved in JavaScript for classes. Needs to be fixed

Reply:
I am on it!